changelogs/fragments/deepcopy-alternative.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- replace copy.deepcopy in high workload areas with a custom function to improve performance (https://github.com/ansible/ansible/pull/44337)
```

Comment:
Do you have bug reports associated with this or timing/memory/etc information?  Link those in the commit message.

Reply:
No, I do not.  The specific instances have largely come from my performance testing, most recently with the threading feature.